pkg/kubelet/server/server_test.go

```diff
@@ -575,6 +576,73 @@ func TestAuthzCoverage(t *testing.T) {
 	}
 }
 
+func TestInstallAuthNotRequiredHandlers(t *testing.T) {
+	fw := newServerTestWithDebug(false, nil)
+	defer fw.testHTTPServer.Close()
+
+	// No new handlers should be added to this list.
+	allowedAuthNotRequiredHandlers := sets.NewString(
+		"/checkpoint/{podNamespace}/{podID}/{containerName}",
```

Comment:
oh wow! checkpoint doesn't require auth?!?

Reply:
Whoops, forgot to update the test case. This was https://github.com/kubernetes/kubernetes/issues/130016